src/app/files/remixd-handle.js

```diff
@@ -55,27 +56,31 @@ export class RemixdHandle extends Plugin {
     * @param {String} txHash    - hash of the transaction
     */
   connectToLocalhost () {
+    let connection = (error) => {
+      if (error) {
+        console.log(error)
+        modalDialogCustom.alert(
+          'Cannot connect to the remixd daemon.' +
+          'Please make sure you have the remixd running in the background.'
```

Comment:
Is it run by default ? Maybe could be good to have a link to documentation.

Reply:
yep, there's already a link in the content of the modal